modules/msk/mutect1/tests/main.nf.test

```diff
@@ -0,0 +1,49 @@
+nextflow_process {
+
+    name "Test Process MUTECT1"
+    script "../main.nf"
+    process "MUTECT1"
+
+    tag "mutect1"
+
+    
+    test("normal_tumor") {
```

Comment:
we'll want to add another test here for the stub via new testing standards: https://github.com/mskcc-omics-workflows/modules/blob/develop/modules/msk/custom/splitfastqbylane/tests/main.nf.test#L38-L62.

Reply:
<img width="1260" alt="image" src="https://github.com/mskcc-omics-workflows/modules/assets/152219469/8a9bf910-c533-4825-b801-3e5ad7fdb805">

Reply:
b47fc44c5a34106a56915b888207438b5f2877af